README.md

```diff
@@ -50,3 +50,164 @@ FAQ
 ---
 
 Go to the [FAQ](https://github.com/Elao/symfony-standard/wiki/FAQ)
+
+Install ElaoStrap (optionnal)
+---
+
+Install our custom twitter bootstrap Symfony bundle : [ElaoStrap](https://github.com/Elao/ElaoThemeElaoStrapBundle)
+    
+Update your /package.json. Here an example :
```

Comment:
here IS an exemple: | here's an exemple:
Il n'y a pas d'espace avant la ponctuation en anglais (? : ; !)